kolibri/core/assets/test/vue.download-button.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-env mocha */
+// The following two rules are disabled so that we can use anonymous functions with mocha
+// This allows the test instance to be properly referenced with `this`
+/* eslint prefer-arrow-callback: "off", func-names: "off" */
+
+'use strict';
+
+const Vue = require('vue-test');
+const downloadButton = require('../src/vue/content-renderer/download-button.vue');
+const DownloadButtonComponent = Vue.extend(downloadButton);
+const assert = require('assert');
+const simulant = require('simulant');
+
+describe('download-button Component', function () {
+  const samplesFiles = [{
+    file_size: 100000,
+    preset: 'High Resolution',
+    download_url: '/downloadcontent/3893fd801427402ad07487c5d2d35119.mp4/Fractions_High_Resolution.mp4',
+  }, {
+    file_size: 500,
+    preset: 'Thumbnail',
+    download_url: '/downloadcontent/187598e1f4596bf4492f5a205922b633.jpg/Fractions_Thumbnail.jpg',
```

Comment:
these long URLs are causing lint errors:

```

01 02 2017 18:28:02.193:ERROR [preprocessor.eslint]: 
1 error(s) in /Users/d/Projects/learning_equality/repos/kolibri/kolibri/core/assets/test/vue.download-button.js
   - 18:1 Line 18 exceeds the maximum line length of 100. Rule: max-len
```

I assume we can shorten them to fake URLs like this?

```javascript
  const samplesFiles = [{
    file_size: 100000,
    preset: 'High Resolution',
    download_url: '/somemovie.mp4',
  }, {
    file_size: 500,
    preset: 'Thumbnail',
    download_url: '/somejpg.jpg',
  }];
```